Code/autopkglib/PathDeleter.py

```diff
@@ -40,7 +40,7 @@ class PathDeleter(Processor):
     def main(self):
         # if recipe writer gave us a single string instead of a list of strings,
         # convert it to a list of strings
-        if isinstance(self.env["path_list"], basestring):
+        if isinstance(self.env["path_list"], str):
             self.env["path_list"] = [self.env["path_list"]]
```

Comment:
No, this changes the behavior in Python 2.
`isiinstance(foo, basestring` is true if foo is a str or if it is unicode.

Your change would be false if self.env["path_list"] is unicode.

Reply:
I'm going to punt the issue down the line for now. I've switched it back to `basestring` until some better testing can be developed.